packages/console/src/components/FormCard/FormCard/index.tsx

```diff
@@ -0,0 +1,50 @@
+import type { AdminConsoleKey } from '@logto/phrases';
+import type { ReactNode } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import DynamicT from '@/ds-components/DynamicT';
+import TextLink from '@/ds-components/TextLink';
+
+import FormCardLayout from '../components/FormCardLayout';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  title: AdminConsoleKey;
+  description?: AdminConsoleKey;
+  learnMoreLink?: string;
+  children: ReactNode;
+};
+
+function FormCard({ title, description, learnMoreLink, children }: Props) {
```

Comment:
The component name is redundant. Consider renaming it to `FormCardLayout`

Reply:
Ah, why do you extract this component again? Don't quite get it

Reply:
updated.